docs/source/bibliography.md

```diff
@@ -52,4 +50,4 @@ IDEs and browsers for my whole academic life.
   Droettboom, Erik Bray, Tom Aldcroft, Matt Davis et al. \"Astropy: A
   community Python package for astronomy.\" *Astronomy & Astrophysics*
   558 (2013): A33.
-  [DOI:10.1051/0004-6361/201322068](http://dx.doi.org/10.1051/0004-6361/201322068)
+  [DOI:10.3847/1538-3881/aabc4f](https://iopscience.iop.org/article/10.3847/1538-3881/aabc4f)
```

Comment:
The DOI is correct, but the title of the paper changed.

Reply:
I can't get where the title is on the website. Is this the title?

```
The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package*
```

Reply:
Or I have to give this title


The Astropy Collaboration, A. M. Price-Whelan, B. M. Sipőcz, H. M. Günther, P. L. Lim, S. M. Crawford, S. Conseil, D. L. Shupe, M. W. Craig, N. Dencheva "The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package"  *The Astropy Collaboration et al  AJ 156 123*(2018)

Reply:
The second one